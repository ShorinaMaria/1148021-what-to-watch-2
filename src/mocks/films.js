const films = [
  {
    title: `Fantastic Beasts: The Crimes of Grindelwald`,
    img: `fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  },
  {
    title: `Pulp fiction`,
    img: `pulp-fiction.jpg`,
  },
  {
    title: `Snatch`,
    img: `snatch.jpg`,
  },
  {
    title: `Moonrise kingdom`,
    img: `moonrise-kingdom.jpg`,
  },
  {
    title: `Macbeth`,
    img: `macbeth.jpg`,
  }
];

export default films;
