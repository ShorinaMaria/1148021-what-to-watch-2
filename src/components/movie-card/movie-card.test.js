import React from 'react';
import renderer from 'react-test-renderer';
import films from '../../mocks/films';
import MovieCard from './movie-card';

it(`renders movie card correctly`, () => {
  const film = films[0];

  const tree = renderer.create(
      <MovieCard
        movie={film}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
