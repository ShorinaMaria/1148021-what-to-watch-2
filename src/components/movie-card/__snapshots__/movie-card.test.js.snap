// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders movie card correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <video
      height="175"
      muted={true}
      poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
      width="280"
    >
      <source
        src="https://gcs-vimeo.akamaized.net/exp=1573000788~acl=%2A%2F955222150.mp4%2A~hmac=1accc82b8704acf030feeca7f5b6ac09298c895bccf94e693e0d349fe53cf23c/vimeo-prod-skyfire-std-us/01/1982/10/259911724/955222150.mp4"
      />
    </video>
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      Fantastic Beasts: The Crimes of Grindelwald
    </a>
  </h3>
</article>
`;
